lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.tsx

```diff
@@ -1,30 +1,28 @@
-import { Link } from '@hypothesis/frontend-shared';
+import { Link } from '@hypothesis/frontend-shared/lib/next';
```

Comment:
This is the only change required to move to the non-deprecated components on frontend-shared, so I took the opportunity to do it here.